src/App.js

```diff
@@ -1,13 +1,26 @@
-import './App.css';
-import Header from './components/Header/Header';
-import Footer from './components/Footer/Footer';
-import Homepage from './components/Homepage/Homepage';
+import { useState } from "react";
+import "./App.css";
+import Header from "./components/Header/Header";
+import Footer from "./components/Footer/Footer";
+import Homepage from "./components/Homepage/Homepage";
+import ProductsPage from "./components/ProductsPage/ProductsPage";
 
 function App() {
+  const [isHomepage, setIsHomepage] = useState(true);
+
   return (
     <div className="App">
-      <Header />
-      <Homepage/>
+      <Header setIsHomepage={setIsHomepage} />
+      {isHomepage ? <Homepage /> : null}
```

Comment:
an easier way to do this is
```suggestion
      {isHomepage && 
      <> 
            <Homepage />
            <button
              className="btn-view-all"
              onClick={() => setIsHomepage(!isHomepage)}
            >
              View All Products
            </button>
      </>

      }
```

Reply:
You're right, and looks way nicer :P